docs/iris/src/developers_guide/pulls.rst

```diff
@@ -1,7 +1,7 @@
 .. _pr_check:
```

Comment:
Changes proposed to this file are the minimum to bring the check list up to date. I expect them to be soon over-written by @tkknight 's wholesale re-write of the Developer's Guide - [see this comment](https://github.com/SciTools/iris/pull/3838#discussion_r487403532)